pulp_python/app/migrations/0003_auto_20200611_2202.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 2.2.3 on 2020-06-11 22:02
```

Comment:
I think we should just go ahead and recreate the migrations entirely -- that is, delete them all, and then run ```python manage.py makemigrations``` again.

This plugin is still pre-release so it doesn't matter if it's a breaking change - and it will be breaking anyway without a data migration.